app.js

```diff
@@ -1,81 +1,132 @@
-const AddBookBtn = document.querySelector('.addBook');
-const bookTitle = document.querySelector('#title');
-const bookAuthor = document.querySelector('#author');
-const booksContainer = document.querySelector('.booksContainer');
-
-// Decalre bookCollections array
-let bookCollections = JSON.parse(window.localStorage.getItem('books')) || [];
-
-// Decalare uniqueId for each books
-let count = 0;
-function uniqueId() {
-  /* eslint-disable no-plusplus */
-  const id = count++;
-  return `book_${id}`;
-}
-
-// Decalre a addBooks function, push the value of title and author in the bookCollection array
-function addBooks() {
-  if (bookTitle.value !== '' && bookAuthor.value !== '') {
-    bookCollections.push({
-      id: uniqueId(),
-      title: bookTitle.value,
-      author: bookAuthor.value,
-    });
+// book class : represents a book
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
   }
-}
 
-// Decalre a clearInput function, to clear the input value unpon clicking the button
-function clearInput() {
-  bookTitle.value = '';
-  bookAuthor.value = '';
-}
+  // UI class handle UI tasks
+  static displayBooks() {
+    // const books = Store.getBooks();
 
-// Save bookCollections to localstorage
-function saveBooks() {
-  window.localStorage.setItem('books', JSON.stringify(bookCollections));
-}
+    const books = JSON.parse(localStorage.getItem('books'));
 
-//  Added Collection of books part.
-function renderBooks() {
-  const storedBooks = JSON.parse(window.localStorage.getItem('books'));
+    Book.addBookTolList(books);
+  }
 
-  if (storedBooks) {
-    const displayBook = storedBooks.map(
+  static addBookTolList(books) {
+    // const list = document.querySelector('#book-list');
+    const row = document.querySelector('.booksContainer');
+    const bookList = books.map(
       (book) => `
-              <article class="d-flex flex-row justify-content-between pb-3 border-bottom">
+            <article class="d-flex flex-row justify-content-between border-bottom pb-3 mb-3">
+            <h2>${book.title}</h2>
+            <h4>${book.author}</h4>
+            <button onclick="Book.removeBook('${book.id}')" class="btn btn-danger" >remove</button>
+            </article>
+    `,
+    );
 
-              <h2> ${book.title} </h2>
-              <p> ${book.author} </p>
+    row.innerHTML = bookList.join('');
 
-              <button onclick="removeBook('${book.id}')" class="removeBtn"> Remove </button>
+    // list.appendChild(row);
+  }
 
-              </article>
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert alert-${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('.container');
+    const form = document.querySelector('#book-form');
+    container.insertBefore(div, form);
 
-              `,
```

Comment:
![image](https://github.com/AbuTalha3/awesomeBooksList/assets/93039359/d96c1c02-6d3d-4855-83fc-4f3df7dd3b29)

**[OPTIONAL]**

- [ ] It would be nice if you put the right name of your project in the readme file to make your readme descriptive and professional 👍